Godeps/Godeps.json

```diff
@@ -545,8 +545,8 @@
 		},
 		{
 			"ImportPath": "github.com/coreos/rkt/api/v1alpha",
-			"Comment": "v1.4.0",
-			"Rev": "f5b69782d120a7ae5ada8f2ebad97a014a98ebd5"
+			"Comment": "v1.6.0",
```

Comment:
#25656 makes that change as well, so this'll cause a merge conflict and, assuming that one gets merged, this change will just be rebased out. If this one goes in first, that can be fine too I think.